firstvoices/backend/views/base_views.py

```diff
@@ -130,6 +135,18 @@ def list(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    def handle_exception(self, exc):
```

Comment:
Currently this block of code seems necessary as the built-in drf framework sets NotAuthenticated calls to 403 when missing an auth header. There is some more drf code setting 401s to 403s as well but it need further investigation Monday.

See `handle_exception` in `/site-packages/rest_framework/views.py` -> `APIView`